lib/foreman_maintain/hammer.rb

```diff
@@ -0,0 +1,37 @@
+module ForemanMaintain
+  class Hammer
```

Comment:
Oh, I see the feature is there: I think it should also provide the methods that we now define here (unless some methods are too generic that could be used from other places). The idea is the lib will contain only very generic code, that would be version independent.